presto-spi/src/main/java/io/prestosql/spi/type/CharType.java

```diff
@@ -78,9 +79,9 @@ public Object getObjectValue(ConnectorSession session, Block block, int position
         }
 
         StringBuilder builder = new StringBuilder(length);
-        String value = block.getSlice(position, 0, block.getSliceLength(position)).toStringUtf8();
-        builder.append(value);
-        for (int i = value.length(); i < length; i++) {
```

Comment:
Not sure it matters, but you can use `value.codePointCount(0, value.size())`.

Reply:
Nice! I didn't know.
I will stay with Slice-provided utility, as we use this in other places as well (here we are fortunate to have a `slice` at hand)

Reply:
@dain thanks again. It was useful -- https://github.com/prestosql/presto/pull/195/files#diff-39912e87f91ce06e350d35bc330d85ceR74